Assets/LeapMotion/Scripts/Animation/TransformTweenBehaviour.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Animation {
+
+  public class TransformTweenBehaviour : MonoBehaviour {
```

Comment:
Would also be nice (if we are trying to make it a script we want devs to use) to either add in some warning on how this script itself doesn't do anything, or we allow it to do stuff :D